.github/workflows/release.yml

```
@@ -2,7 +2,7 @@ name: Release (release-please, multi-language assets)
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, dev ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)